<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1136</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="BtnLoad">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>260</y>
      <width>200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDamage">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>340</y>
      <width>200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>Damage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFix">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>420</y>
      <width>200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>Fix</string>
    </property>
   </widget>
   <widget class="QLabel" name="LblSource">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>391</width>
      <height>170</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Source Image</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LblDamaged">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>391</width>
      <height>170</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Damages</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="LblFixed">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>391</width>
      <height>170</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Fixed Image</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="SpinerDamSize">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>70</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>40</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="SliderDamNum">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>150</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>40</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="SpinDamSize">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>70</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>40</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="SpinDamNum">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>150</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>40</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Damages Size</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>130</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Num of Damage</string>
    </property>
   </widget>
   <widget class="QLabel" name="LblMap">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>391</width>
      <height>170</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Map</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="getMap">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>500</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Map</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1136</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>SpinerDamSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SpinerDamSize</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SpinerDamSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SpinDamSize</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>599</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>685</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliderDamNum</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SpinDamNum</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>687</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SpinDamNum</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SliderDamNum</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>665</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>598</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SpinDamSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SpinerDamSize</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>672</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>609</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
